user_guide_src/source/changelogs/v4.6.0.rst

```diff
@@ -99,6 +99,16 @@ See :ref:`Upgrading Guide <upgrade-460-sid-change>` for details.
 
 .. _v460-interface-changes:
 
+Headers
+-------
+
+The headers set by the ``Response`` class replace those that can be set by the PHP
+``header()`` function.
+
+In previous versions, headers set by the ``Response`` class were added to existing
+ones - giving no options to change them. That could lead to unexpected behavior when
+the same headers were set with mutually exclusive directives.
```

Comment:
Can you elaborate on this "mutually exclusive headers"?

Reply:
I've added some additional explanations below, in the changelog.